pkg/features/features.go

```diff
@@ -74,6 +74,12 @@ const (
 	// owner: @hown3d
 	// alpha: v1.130.0
 	UseUnifiedHTTPProxyPort featuregate.Feature = "UseUnifiedHTTPProxyPort"
+
+	// VPAInPlaceUpdates enables the usage of in-place Pod resource updates in the Vertical Pod Autoscaler deployments.
+	// resources to perform in-place Pod resource updates.
+	// owner: @vitanovs @ialidzhikov
+	// alpha: v1.128.0
```

Comment:
Let's target v1.132.0:

```suggestion
	// alpha: v1.132.0
```

Reply:
Thanks for the suggestion, updated the version to `v1.132.0` in my latest updates :))